documentation/library_guide/random.rst

```diff
@@ -72,6 +72,13 @@ Distribution                   Description
 ``uniform_int_distribution``   Produces integer values evenly distributed across a range
 ``uniform_real_distribution``  Produces real values evenly distributed across a range
 ``normal_distribution``        Produces real values according to the Normal (Gaussian) distribution
+``exponential_distribution``   Produces real values according to the Exponential distribution
+``bernoulli_distribution``     Produces boolean values: 1 with probability p, and 0 with probability (1-p)
```

Comment:
Should it be true/false?

Reply:
I changed the description to different one.